### PR TITLE
Refactor: add adelais rev.3

### PR DESCRIPTION
**Pull request by mechlovin**
## Description

Add Adelais rev.3 mono backlight (standard_led). 
Same matrix, layout, pinout. Just add 3 rotary encoder.
Have nice day, Thanks!

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mechlovin/adelais/standard_led/config.h [hunk 1]
```diff
@@ -13,3 +13,8 @@
 #define RGBLED_NUM 23
 #define RGBLIGHT_LIMIT_VAL 255
 #define RGBLIGHT_ANIMATIONS
+
+#define ENCODERS_PAD_A { A6, A4, B7 }
+#define ENCODERS_PAD_B { A5, A3, B6 }
+#define ENCODER_RESOLUTION 4
```
Comment (Erovia) [inline keyboards/mechlovin/adelais/standard_led/config.h:19]: Wouldn't it make more sense to move this to `keyboards/mechlovin/adelais/standard_led/rev3/config.h`?  
Therefore the feature is not enabled for everybody.

File: keyboards/mechlovin/adelais/standard_led/rules.mk [hunk 2]
```diff
@@ -1,3 +1,4 @@
 BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
 BACKLIGHT_DRIVER = pwm
-RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
\ No newline at end of file
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+ENCODER_ENABLE = yes
```
Comment (Erovia) [inline keyboards/mechlovin/adelais/standard_led/rules.mk:4]: Wouldn't it make more sense to move this to `keyboards/mechlovin/adelais/standard_led/rev3/rules.mk`?  
Therefore the feature is not enabled for everybody.

File: keyboards/mechlovin/adelais/standard_led/config.h (hunk 1 above)
Comment (mechlovin) [reply keyboards/mechlovin/adelais/standard_led/config.h]: thanks.
I just reread the tutorial about sub_folder 😄

File: keyboards/mechlovin/adelais/standard_led/rev3/rules.mk [hunk 3]
```diff
@@ -0,0 +1,4 @@
+ENCODER_ENABLE = yes
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
+BACKLIGHT_DRIVER = pwm
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
```
Comment (Erovia) [inline keyboards/mechlovin/adelais/standard_led/rev3/rules.mk:4]:
```suggestion

```

File: keyboards/mechlovin/adelais/standard_led/rev2/rules.mk [hunk 4]
```diff
@@ -0,0 +1,3 @@
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
+BACKLIGHT_DRIVER = pwm
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
```
Comment (Erovia) [inline keyboards/mechlovin/adelais/standard_led/rev2/rules.mk:3]:
```suggestion

```
Just leave this file empty.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**